src/menu.css.ts

```diff
@@ -1,40 +1,41 @@
-import {style, createVar} from '@vanilla-extract/css';
+import {createVar, style} from '@vanilla-extract/css';
 import {sprinkles} from './sprinkles.css';
 import {vars as colorsVars} from './skins/skin-contract.css';
 
 const top = createVar();
 const bottom = createVar();
-const right = createVar();
+const left = createVar();
 const width = createVar();
-const transformOrigin = createVar();
 const maxHeight = createVar();
+const transformOrigin = createVar();
 
 export const vars = {
     top,
     bottom,
-    right,
+    left,
```

Comment:
is this change needed?

Reply:
Yes, because its easier to understand by anyone a left oriented coordinate system.